components/filter/opportunities.tsx

```diff
@@ -140,6 +152,18 @@ const OpportunityDropdown = ({
     return title.replace(/&/g, "%26");
   };
 
+  const appendURLWithId = (id: string) => {
+    const currentURL = window.location.href.split("#")[0];
```

Comment:
There are 2 variables in this file named currentURL that have different values.

Can you name them more appropriately?

Reply:
done